charts/sscs-tya-notif/values.aat.template.yaml

```diff
@@ -2,27 +2,18 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
-    IDAM_API_URL: ${IDAM_URL}
+    CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
```

Comment:
This should be ok to move too values.yaml and follow the `{{ .Values.global.environment }}` pattern. This has worked for other projects (see sscs bulk scan)